packages/components/organisms/f-cookie-banner/package.json

```diff
@@ -47,7 +47,10 @@
   },
   "peerDependencies": {
     "@justeat/browserslist-config-fozzie": ">=1.1.1",
-    "js-cookie": "2.2.1"
+    "js-cookie": "2.2.1",
+    "@justeat/f-button": "3.0.2",
+    "@justeat/f-link": "2.0.0",
+    "@justeat/f-mega-modal": "3.0.2"
```

Comment:
These should all be set as ranges to allow different versions to be used across consuming apps or components.

Reply:
Good spot. 👍

Reply:
Fixed.